cicd/3-app/javabuilder/template.yml.erb

```diff
@@ -669,20 +669,21 @@ Resources:
         OKActions: []
 
 <%JAVALAB_APP_TYPES.each do | name | -%>
-
-  <%=name%>HighSevereErrorRateAlarm:
+<%{
+  TenPercentSevereErrorRateAlarm: {Threshold: 10, AlarmName: 'ten_percent_severe_error_rate'},
+  NinetyPercentSevereErrorRateAlarm: {Threshold: 90, AlarmName: 'ninety_percent_severe_error_rate'},
+}.each do |alarmTitle, config| -%>
+  <%=name%><%=alarmTitle%>:
     Type: AWS::CloudWatch::Alarm
     Properties:
-      AlarmName: !Sub "${SubDomainName}_<%=name.downcase%>_high_severe_error_rate"
-      AlarmDescription: Send page if Javabuilder severe error rate exceeds 10% for 20
-          minutes. Occasional spikes are expected, but a sustained high error rate
-          is an indication of an outage.
-      ActionsEnabled: true
-      AlarmActions:
-          - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:Javabuilder-high-error-rate"]
+      AlarmName: !Sub "${SubDomainName}_<%=name.downcase%>_<%=config[:AlarmName]%>"
+      AlarmDescription: Severe error rate in Javabuilder's <%=name%> build and run lambda (the core of
+          Javabuilder, which executes student <%=name%> code) exceeded <%=config[:Threshold]%>% for four
+          consecutive 5 minute periods.
```

Comment:
Could we also add a bit of clarification here (or in the composite alarm definition below) for the difference between the severe error rate and non-severe? Could just be that severe errors are thrown by our code vs non-severe errors are thrown by the lambda system (ex. timeouts)

Reply:
will do